docs/src/examples/juliatypes.md

```diff
@@ -75,6 +78,69 @@ savefigs("colormap", td) # hide
 
 ![](colormap.svg)
 
+### ggplot2
+
+Something that looks a bit like ggplot2.
+
+```@example pgf
+using Colors
+using LaTeXStrings
+
+ggplot2 = @pgf {
```

Comment:
Perhaps call this variable `ggplot2_options` or `ggplot2_style`?

Reply:
Sure